includes/Hooks.php

```diff
@@ -220,4 +223,17 @@ public static function onImportHandlePageXMLTag( $importer, array &$pageInfo ):
 		// This prevents continuing down the else-if statements in WikiImporter, which would reach `$tag != '#text'`
 		return false;
 	}
+
+	/**
+	 * - Register our database schema.
+	 * - Initialize domain ID.
+	 *
+	 * @see https://www.mediawiki.org/wiki/Manual:Hooks/LoadExtensionSchemaUpdates
+	 *
+	 * @param DatabaseUpdater $updater
```

Comment:
You do not need this comment and can instead put the type in the PHP. If you implement one of those hook interfaces you are forced to use comments, which is a big reason to not use these.

Reply:
I tried moving the type annotation away from the comment, got this error:
```
PHP Fatal error:  Declaration of DataAccounting\Hooks::onLoadExtensionSchemaUpdates(DatabaseUpdater $updater) must be compatible with MediaWiki\Installer\Hook\LoadExtensionSchemaUpdatesHook::onLoadExtensionSchemaUpdates($updater) in /var/www/html/extensions/DataAccounting/includes/Hooks.php on line 234
```
This is because in the MW source code of installer/Hook/LoadExtensionSchemaUpdatesHook.php, the signature of the interface is
```php
interface LoadExtensionSchemaUpdatesHook {
	/**
	 * This hook is called during database installation and updates.
	 *
	 * Do not use this hook with a handler that uses a "services" or "optional_services"
	 * option in its ObjectFactory spec. It is called in a context where the global
	 * service locator is not initialised.
	 *
	 * @since 1.35
	 *
	 * @param DatabaseUpdater $updater DatabaseUpdater subclass
	 * @return bool|void True or no return value to continue or false to abort
	 */
	public function onLoadExtensionSchemaUpdates( $updater );
}
````